Fly-link update to 0.4.0

Fly33 opened the pull request:
+ region selection with the polygon draw tool
+ exclusion of portals with the marker draw tool
+ steady drawing (portals is not excluded from the drawing if they are
moved out of boundaries)

Commit:
Fly-link update to 0.3.0

+ region selection with the polygon draw tool
+ exclusion of portals with the marker draw tool
+ steady drawing (portals is not excluded from the drawing if they are
moved out of boundaries)

modos189 commented:
Looks great. I see no obstacles to merger after the corrective release that will be released the other day.

Commit:
Correct Author name

Because my in-game nick name is Fly33

Commit:
Link suggestion

+ Now you can modify set of links by suggesting links using draw tools
lines. It takes into account only the lines with both ends snapped to the
portals inside the same polygon. The result set of links will contain
the suggested links. If there is no valid set of link you'll
see nothing.

johnd0e commented:
@Fly33 
Could you comment ideas of #203?
Would it be hard to implement?

modos189 commented on plugins/fly-links.js:
```diff
@@ -221,32 +217,168 @@ window.plugin.flyLinks.updateLayer = function() {
       opacity: 1,
       weight: 1.5,
-      interactive: false,
+      clickable: false,
```
```suggestion
      interactive: false,
```

modos189 commented on plugins/fly-links.js:
```diff
@@ -221,32 +217,168 @@ window.plugin.flyLinks.updateLayer = function() {
       fillColor: '#FF0000',
       fillOpacity: 1 - Math.pow(0.85, triangle.depth),
-      interactive: false,
+      clickable: false,
```
```suggestion
      interactive: false,
```

modos189 reviewed (changes requested):
New version of Leaflet uses the keyword *interactive*. Please do not return outdated *clickable*

Commit:
Fix Leaflet 1.x compatibility

Co-authored-by: Alexander Danilov <git@modos189.ru>
